src/group_theory/nilpotent.lean

```diff
@@ -377,3 +377,23 @@ begin
     apply mem_closure.mpr,
     exact λ K hK, hK ⟨f y, hd (mem_map_of_mem f hy), by simp⟩ }
 end
+
+lemma lower_central_series_le_center_to_nilpotent {n : ℕ} (h : lower_central_series G n ≤ center G)
```

Comment:
The name suggests that the conclusion will be `is_nilpotent G`. But you prove something more precise. If you want to keep it that way, please reflect it in the name.